src/org/omegat/core/team2/impl/SVNAuthenticationManager.java

```diff
@@ -232,4 +239,46 @@ public String getProxyUserName() {
         public void acknowledgeProxyContext(boolean accepted, SVNErrorMessage errorMessage) {
         }
     };
+
+    private SVNAuthentication getResult(String kind, SVNURL url, String user, char[] pass)
```

Comment:
A method name `getResult` has not enough meaning. It should be like `getAuthenticatorInstance`.